itou/files/models.py

```diff
@@ -15,3 +18,13 @@ class File(models.Model):
 
     class Meta:
         verbose_name = "fichier"
+
+    @staticmethod
+    def anonymized_filename(filename):
```

Comment:
Attention : si on veut mettre le fichier dans un repertoire (`resume/file.pdf`) to code va supprimer le repertoire.